python/langsmith/testing/_internal.py

```diff
@@ -837,12 +820,6 @@ def _create_test_case(
     )
     example_id, example_name = _get_example_id(func, inputs, test_suite.id)
     example_id = langtest_extra["id"] or example_id
-    test_suite.sync_example(
-        example_id,
-        inputs=inputs,
-        outputs=outputs,
-        metadata={"signature": _get_test_repr(func, signature), "name": example_name},
-    )
```

Comment:
are we sure the example doesn't need to exist before its referenced in a run? also i think if we dont do this here we lose pytest fixture inputs/outputs

Reply:
I think it will work but will test with a fresh dataset too

Reply:
Added dedicated logging for fixture inputs and outputs, and @angus-langchain confirmed this will work